wp-session-manager.php

```diff
@@ -3,33 +3,26 @@
  * Plugin Name: WP Session Manager
  * Plugin URI:  http://jumping-duck.com/wordpress/plugins
  * Description: Prototype session management for WordPress.
- * Version:     1.2.1
+ * Version:     2.0
  * Author:      Eric Mann
  * Author URI:  http://eamann.com
  * License:     GPLv2+
  */
 
-// let users change the session cookie name
-if( ! defined( 'WP_SESSION_COOKIE' ) ) {
-	define( 'WP_SESSION_COOKIE', '_wp_session' );
-}
+require __DIR__ . '/vendor/autoload.php';
```

Comment:
You might consider wrapping this in a `file_exists()` check. If someone were to require this package directly in a site-wide composer manifest, `vendor/` would not be inside this directory.